src/Symfony/Component/VarDumper/Caster/CurlCaster.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class CurlCaster
```

Comment:
let's make all classes final?

Reply:
I’d say internal personally, covering them with BC and given object representations often change from a PHP version to another would be too much of a burden.